scripts/core-triage/project.py

```diff
@@ -0,0 +1,195 @@
+# imports
+import os
+import requests
+
+# TODO: improve
+from graphql_queries import *
+
+# TODO: what is error handling?
+
+# constants # TODO: move to a config file (or something) <= copilot suggested this last part
+ORG = "dbt-labs"
+PROJECT_NAME = "Core triage"
+PROJECT_NUM = 22
+# list of Core teams
+CORE_TEAMS = ["core"]
+# read newline separated list of projects from file
+REPOS = [
```

Comment:
yes I love it. I'm fairly confident `dbt-server` is failing because it's internal and the PAT doens't have scope for that (whereas the local one I use for testing does), so in short any non-public repos for this action will fail. which I also think is fine, because this is for our OSS triage so everything should be public anyway

changing this over